### PR TITLE
[CRN-1007] Shared Outputs amount

### PR DESCRIPTION
**Pull request by evemontalvao**

### CODE REVIEW

**Review by oratiu — APPROVED** [review]
Looks good

File: packages/react-components/src/templates/TeamProfileHeader.tsx [hunk 1]
```diff
@@ -247,7 +249,9 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
         {tools && (
           <TabLink href={route.workspace({}).$}>Team Workspace</TabLink>
         )}
-        <TabLink href={route.outputs({}).$}>Team Outputs</TabLink>
+        <TabLink href={route.outputs({}).$}>
+          Shared Outputs ({sharedOutputsCount})
```
Comment (kmaid) [inline packages/react-components/src/templates/TeamProfileHeader.tsx:253]: Shared Outputs Count should default to zero as the value is optional?

Comment (evemontalvao) [reply packages/react-components/src/templates/TeamProfileHeader.tsx:253]: yes, you're right! changing it now

File: apps/crn-frontend/src/network/teams/TeamProfile.tsx [hunk 2]
```diff
@@ -74,6 +74,8 @@ const TeamProfile: FC<TeamProfileProps> = ({ currentTime }) => {
   );
 
   if (team) {
+    const sharedOutputsCount = team.outputs?.length || 0;
```
Comment (kmaid) [inline apps/crn-frontend/src/network/teams/TeamProfile.tsx:77]: Could we default this inside the component? It be cleaner

File: packages/react-components/src/templates/TeamProfileHeader.tsx [hunk 3]
```diff
@@ -145,6 +146,7 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
   teamListElementId,
   labCount,
   upcomingEventsCount,
+  sharedOutputsCount,
```
Comment (kmaid) [inline packages/react-components/src/templates/TeamProfileHeader.tsx:149]: here

File: packages/react-components/src/templates/__tests__/TeamProfileHeader.test.tsx [hunk 4]
```diff
@@ -17,6 +17,7 @@ const boilerplateProps: ComponentProps<typeof TeamProfileHeader> = {
   labCount: 15,
   upcomingEventsCount: 0,
   pastEventsCount: 0,
+  sharedOutputsCount: 0,
```
Comment (kmaid) [inline packages/react-components/src/templates/__tests__/TeamProfileHeader.test.tsx:20]: If these are optional props we shouldn't include them in the default props

File: apps/crn-frontend/src/network/teams/TeamProfile.tsx (hunk 2 above)
Comment (evemontalvao) [reply apps/crn-frontend/src/network/teams/TeamProfile.tsx:77]: Sure, I thought you would prefer this way to keep it like the other two counts, where there's no need to use a default value, but changing it

**Pull request merged**